.github/workflows/test_and_deploy.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9]
```

Comment:
The `python-version` specified here is not consistent with `pytests.yml`, namely lacking Python 3.10. Are we supporting for 3.10?

Reply:
Good catch! Fixed.